run-tests.sh

```diff
@@ -87,8 +95,8 @@ for arch in $ARCHS; do
         compiler_rt_arch=i386
     fi
     for i in libc++ libunwind libssp-0 libclang_rt.asan_dynamic-$compiler_rt_arch; do
-        if [ -f $PREFIX/$arch-w64-mingw32/bin/$i.dll ]; then
-            cp $PREFIX/$arch-w64-mingw32/bin/$i.dll $TEST_DIR
+        if [ -f $PREFIX/$arch-w64-$target_os/bin/$i.dll ]; then
```

Comment:
Yes and no. I haven't updated these yet. But in the future it may be necessary. For example right now winpthreads doesn't pass the WACK because it's using forbidden APIs. Depending on how we fix it we may need different built versions, the UWP one with less calls possible or doing errors in some calls.
And in the long term it would be safer to build UWP with libraries that only contain allowed APIs. This way using forbidden APIs will fail to link. That's also what happens in some cases with MSVC.

But indeed, for now this line is wrong.

Reply:
Hmm, right. For most cases, static libs where you optionally can link in winstorecompat (to avoid linking to the forbidden functions) should work with both uwp and the full desktop API. But I do see the value in separate tailored libs for uwp/winrt in the future though.